.github/workflows/acceptance.yml

```
@@ -615,8 +615,8 @@ jobs:
           browser: chrome
           spec: cypress/tests/coresandbox/**/*.js
           start: |
-            make start-test-acceptance-server-coresandbox
-            make start-test-acceptance-frontend-coresandbox
+            make coresandbox-acceptance-backend-start
```

Separate words with kebab case.

```suggestion
            make core-sandbox-acceptance-backend-start
```

---

Decided to leave it as it is :)